src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -189,6 +193,37 @@ private function getDocBlock($class, $property)
         return $this->docBlocks[$propertyHash] = $data;
     }
 
+    /**
+     * Gets the DocBlock from a constructor.
+     *
+     * @param string $class
+     * @param string $property
+     *
+     * @return DocBlock|null
+     */
+    private function getDocBlockFromConstructor($class, $property)
+    {
+        try {
+            $reflectionClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+        $reflectionConstructor = $reflectionClass->getConstructor();
+        if (!$reflectionConstructor) {
+            return null;
+        }
+        if (!preg_match('/\@param.+?\$'.$property.'/', $reflectionConstructor->getDocComment(), $matches)) {
```

Comment:
Done. Now I parse the phpdoc params using reflection-docblock, then filter docblock.tags so that only the requested property param is kept and then recreate the docblock object. The code became more verbose though.